<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="myTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="revertAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newAction"/>
   <addaction name="openAction"/>
   <addaction name="saveAction"/>
   <addaction name="saveAsAction"/>
   <addaction name="revertAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockCalc">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>105</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>105</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Calculator</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="myInput"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="plusButton">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="minusButton">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="multButton">
           <property name="text">
            <string>*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="divideButton">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="newAction">
   <property name="icon">
    <iconset resource="task8.qrc">
     <normaloff>:/images/filenew.png</normaloff>:/images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="task8.qrc">
     <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="task8.qrc">
     <normaloff>:/images/filesave.png</normaloff>:/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="icon">
    <iconset resource="task8.qrc">
     <normaloff>:/images/filesaveas.png</normaloff>:/images/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="revertAction">
   <property name="icon">
    <iconset resource="task8.qrc">
     <normaloff>:/images/revert.png</normaloff>:/images/revert.png</iconset>
   </property>
   <property name="text">
    <string>Revert</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="task8.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>newAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAsAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>revertAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>revert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>myTextEdit</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>documentWasModified()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plusButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>plusPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minusButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>minusPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>multButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>multPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>divideButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>dividePressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newFile()</slot>
  <slot>open()</slot>
  <slot>save()</slot>
  <slot>saveAs()</slot>
  <slot>revert()</slot>
  <slot>documentWasModified()</slot>
  <slot>plusPressed()</slot>
  <slot>minusPressed()</slot>
  <slot>multPressed()</slot>
  <slot>dividePressed()</slot>
 </slots>
</ui>
